app/helpers/batch_edits_helper.rb

```diff
@@ -1,14 +0,0 @@
-# View Helpers for Hydra Batch Edit functionality
-module BatchEditsHelper
```

Comment:
Good question.  Perhaps the answer depends on an answer to the opposite question, i.e., why was it NOT in the hyrax namespace?  It could be a separate PR, but this PR is adding what seems to be related batch-select code, which is working when it's in the hyrax namespace.  To be consistent, it looks like the batch-* stuff can live in the hyrax namespace.  But, I dunno, /shrug